### DISCUSSION TITLE
Can't create new user to SO, Kibana, etc.

### DISCUSSION THREAD

**Discussion by vbrothanek** [Security-Onion-Solutions/securityonion#9088] [Unsupported Versions] [answered]
Hello!

I have a problem with adding a new user to security onion, kibana, etc.
I am adding the new user from the command line using `sudo so-user-add email@address.com`.
Then I get the message: **Email addresses cannot contain uppercase letters**.

I'm using SO version 2.3.181 on CentOS 7.

Does anyone else have this problem or am I doing something wrong?

Thanks for any help.
Vojtech

**Comment by dougburks** [comment, marked as answer]
Are you trying to use any uppercase letters?

Here's how that check is performed:
https://github.com/Security-Onion-Solutions/securityonion/blob/484aa7b2075eb7abb8c53cf021a90d1bd6bf8211/salt/common/tools/sbin/so-user#L118-L120

**Reply by vbrothanek** [reply]
Hello! 

Thank you for response, no I did not use any capital letters.. I even tried to enter the mail from the manual for a test..
`sudo so-user-add tom@example.com`

link: https://docs.securityonion.net/en/2.3/adding-accounts.html

The output was the same.

Best Regards, Vojtech.

**Reply by dougburks** [reply]
I just tested and this works correctly for me:
```
sudo so-user-add tom@example.com
Enter new password:
Syncing users and roles between SOC and Elastic...
Elastic state will be re-applied to affected minions. This may take several minutes...
Successfully added new user to SOC
```

As you can see in the code, this is a very simple check for uppercase letters:
https://github.com/Security-Onion-Solutions/securityonion/blob/484aa7b2075eb7abb8c53cf021a90d1bd6bf8211/salt/common/tools/sbin/so-user#L118-L120

What is your keyboard layout set to?

Have you tried a different keyboard?

Have you tried a different computer?

**Reply by vbrothanek** [reply]
Hi, 

Thank you for the feedback, I use the keyboard with English layout. But for some reason I couldn't create a new user by connecting to the server via SSH, when I connected directly via the console on the virtual machine it worked fine. 

Thank you again for your help.
Have a nice day.
Vojtech

**Reply by tswiderski1** [reply]
I have the same problem. When I connect via ssh, I can't add a user

**Comment by tswiderski1** [comment]
I noticed that the problem occurs when I connect from a linux terminal. When I connected via putty, the problem did not occur.
